src/index.html

```diff
@@ -6,7 +6,42 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <section class="calendar calendar--start-date--sun calendar--month-length--31">
```

Comment:
```suggestion
      <div class="calendar calendar--start-date--sun calendar--month-length--31">
```

I suggest to use <div> here instead of <section>

> The section tag is used when requirements of two headers or footers or any other section of documents needed. Section tag grouped the generic block of related contents. The main advantage of the section tag is, it is a semantic element, which describes its meaning to both browser and developer.

You can read more about it in [Here](https://www.geeksforgeeks.org/html-section-tag/#:~:text=The%20section%20tag%20is%20used,to%20both%20browser%20and%20developer.)

Reply:
Thank you Saleh. Changes are implemented.

Reply:
Happy to help 😅